jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/MultiPartCaptureTest.java

```diff
@@ -60,6 +60,7 @@ public void testMultiPartFormDataParse(MultiPartRequest formRequest, Charset def
         MultiPartFormData.Parser parser = new MultiPartFormData.Parser(boundary);
         parser.setUseFilesForPartsWithoutFileName(false);
         parser.setFilesDirectory(tempDir);
+        parser.setUseFilesForPartsWithoutFileName(true);
```

Comment:
Huh?

Why set it to false, then to true?

Reply:
Good point, didn't see this line already there.

It needs to be set to `true` not `false`, because there is a default `maxMemoryFileSize` of `0`. So if we have `useFilesForPartsWithoutFileName == false` then we can never store any part with no name in memory as it will exceed the `maxMemoryFileSize`.

We could also get these tests to pass with something like this
```java
parser.setUseFilesForPartsWithoutFileName(false);
parser.setMaxMemoryFileSize(1024 * 16);
```